src/LoadImage.js

```diff
@@ -46,6 +46,7 @@ function LoadImage(ref, name, src, main_callback) {
   function loadImage(name, src) {
     var step = {
       name: "load-image",
+      description: "This initial step loads and displays the original image without any modifications.",
```

Comment:
This is because this `loadImage` special module does not have an `info.json`, but maybe it should.

Reply:
Perhaps a type of "inputModules" distinct from regular modules is a good idea. This could include for example a module for capturing frames from video, a module for capturing from the webcam, and other ways of initiating a sequence.

Reply:
Yes... That's right; Although we did make the load-image module a lot like the other modules :-) But I think it might be best to keep it (and other such modules) on their own